buttons.py

```diff
@@ -0,0 +1,29 @@
+from PyQt5.QtWidgets import QPushButton
```

Comment:
Hi JeanExtreme002, 
this is the first pull request I ever receive, it's kind of exciting.
You will see a few questions in the code because there are some things that are not clear to me. It is my fault, though: I am a mechanical engineer currently doing a Ph.D. in Robotics: I am a self-taught software developer, just for fun. 
So, probably most of my questions will seem stupid. Please, be patient :D. 
Thank you for your contribution, I really appreciated.